docusaurus/docs/getting-started/how-to-make-a-graph.md

```diff
@@ -0,0 +1,90 @@
+---
+id: how-to-make-a-graph
+title: 'Making a Graph'
+tags: ['introduction', 'how-to', 'getting started']
+---
+
+### Requirements
+* C++17 or later
+* GCC compiler v7.3.0 OR MSVC compiler supporting C++17 or later
+
+## Creating the Graph
+
+At the heart of CXXGraph is the graph object. The graph object contains the nodes and edges to be worked upon.
+
+Syntax for creating the graph object:
+
+```
+Graph<int> myGraph; 
+```
+
+Whether a graph is directed or undirected is controlled by the edges added, *not the graph object itself.*
+
+### Adding Nodes and Edges
+
+To begin using the graph, we require nodes to create edges between. Nodes are created individually.
+
+Here we create two nodes...
+
+```
+CXXGRAPH::Node<int> node1("1", 1);
+CXXGRAPH::Node<int> node2("2", 2);
```

Comment:
```
CXXGRAPH::Node<int> node1("1", 1);
CXXGRAPH::Node<int> node2("2", 2);
```

Should be

```
CXXGraph::Node<int> node1("1", 1);
CXXGraph::Node<int> node2("2", 2);
```

All the references below should change `CXXGRAPH` -> `CXXGraph`

Reply:
Yes this is true, @Horicabu  can you correct these?